L1Trigger/L1TTrackMatch/python/L1TrackSelectionProducer_cfi.py

```diff
@@ -26,24 +26,24 @@
   debug = cms.int32(0) # Verbosity levels: 0, 1, 2, 3, 4
 )
 
-L1TrackSelectionProducerExtended = L1TrackSelectionProducer.clone(
-  l1TracksInputTag = ("L1GTTInputProducerExtended","Level1TTTracksExtendedConverted"),
+l1tTrackSelectionProducerExtended = l1tTrackSelectionProducer.clone(
+  l1TracksInputTag = ("l1tGTTInputProducerExtended","Level1TTTracksExtendedConverted"),
   outputCollectionName = "Level1TTTracksExtendedSelected",
-  cutSet = cms.PSet(
-                    ptMin = cms.double(3.0), # pt must be greater than this value, [GeV]
-                    absEtaMax = cms.double(2.4), # absolute value of eta must be less than this value
-                    absZ0Max = cms.double(15.0), # z0 must be less than this value, [cm]
-                    nStubsMin = cms.int32(4), # number of stubs must be greater than or equal to this value
-                    nPSStubsMin = cms.int32(0), # the number of stubs in the PS Modules must be greater than or equal to this value
-
-                    reducedBendChi2Max = cms.double(2.4), # bend chi2 must be less than this value
-                    reducedChi2RZMax = cms.double(10.0), # chi2rz/dof must be less than this value
-                    reducedChi2RPhiMax = cms.double(40.0), # chi2rphi/dof must be less than this value
-
-                    deltaZMaxEtaBounds = cms.vdouble(0.0, 0.7, 1.0, 1.2, 1.6, 2.0, 2.4), # these values define the bin boundaries in |eta|
-                    deltaZMax = cms.vdouble(3.0, 3.0, 3.0, 3.0, 3.0, 3.0), # delta z must be less than these values, there will be one less value here than in deltaZMaxEtaBounds, [cm]
+  cutSet = dict(
+                    ptMin = 3.0, # pt must be greater than this value, [GeV]
+                    absEtaMax = 2.4, # absolute value of eta must be less than this value
```

Comment:
Minor point: to simplify the configuration one could modify only those parameters in the PSet that change with respect to the cloned ones, and remove the other ones from here

Reply:
Minor point: to simplify the configuration one could modify only those parameters in the PSet that change with respect to the cloned ones, and remove the other ones from here